base/scripts/objectives.cs

```diff
@@ -947,6 +947,15 @@ function checkObjectives(%this)
 
       //active code
       zadmin::ActiveMessage::All(FlagReturned, %flagTeam, %playerClient);
+      
+      //midair return stat
+      if(!Player::ObstructionsBelow(%playerClient, $Game::Midair::Height))
+        {
+
+            zadmin::ActiveMessage::All(FlagInterception, %flagTeam, %playerClient);
+            
+        }
+
```

Comment:
formatting nit:

```suggestion
      //midair return stat
      if(!Player::ObstructionsBelow(%playerClient, $Game::Midair::Height)) {
            zadmin::ActiveMessage::All(FlagInterception, %flagTeam, %playerClient);
      }
```

Reply:
im a stickler on white space too however when in the creation phase i like to leave a lot of space open to let me know an area ive been working on, ideally once the code is at a good spot, i can go back and clean a lot of white space up

Reply:
given that we're at code merge time, this is an appropriate time to clean up things like this. i don't feel like asking to clean up whitespace is an unfair request

Reply:
ill make a habit of it